extension/test/e2e/credit-card-3ds-native.spec.js

```diff
@@ -182,8 +182,6 @@ describe('::creditCardPayment3dsNative::', () => {
       makePaymentResponse
     )
 
-    await browserTab.waitForTimeout(15_000)
```

Comment:
I removed this as in the next line in the method `await creditCardNativePage.finish3dsNativePayment()` we wait again. So we had 2x waitForTimeout, which is redundant.